cast/js/src/main/resources/preamble.js

```diff
@@ -133,21 +133,26 @@ DOMNode = function DOMNode() { // An impostor for the Node class
         
         this.selectNodes = function(a) {
 	}
+
+	this.hasAttribute = function(a) {
+	}
 };
 
-DOMNode.prototype.addEventListener = function Node_prototype_addEventListener(name, fn) { fn(); };
+DOMNode.prototype.addEventListener = function Node_prototype_addEventListener(name, fn, options) { fn(); };
```

Comment:
Why do we need this change?  Is some call edge missing if we don't do it?  It's an ok change, just wondering why it's needed

Reply:
These changes don't impact the call edge. As I was updating models, I updated the signatures for some.